packages/xrpl/test/models/utils.test.ts

```diff
@@ -71,8 +71,8 @@ describe('Models Utils', function () {
       const { tfPassive, tfFillOrKill } = OfferCreateFlags
       const expected: number = tfPassive | tfFillOrKill
 
-      setTransactionFlagsToNumber(tx)
-      assert.strictEqual(tx.Flags, expected)
+      const flagNum = convertTxFlagsToNumber(tx)
```

Comment:
I would leave the `setTransactionFlagsToNumber` in as well, until it gets removed. Just to ensure there's no other breaking changes.

Reply:
Issue here is that the tests now fail because `setTransactionFlagsToNumber` was marked as deprecated. Is there a way to override this?

Reply:
@mvadari Why should we retain `setTransactionFlagsToNumber` inside a test file? How will that cause a breaking change?

Reply:
Figured out the test failure issue, I agree with @mvadari here. Since its still in use and is modified, we would want to make sure the expected behavior didn't changed (even though its deprecated now)